docker/README.md

```diff
@@ -68,7 +68,7 @@ docker run \
   openziti/ziti-host
 ```
 
-This example uses [the included Docker Compose project](./docker-compose.yml) to illustrate publishing a server container to your OpenZiti Network.
+This example uses [the included Docker Compose project](https://github.com/openziti/ziti-tunnel-sdk-c/blob/main/docker/docker-compose.yml) to illustrate publishing a server container to your OpenZiti Network.
```

Comment:
i don't like these changes. using the full url in this way doesn't feel worth it

Reply:
It's essential for including this README in the Docusaurus site because the build doesn't know about relative links.

Reply:
@dovholuknf and I found that Docusaurus will in fact render relative links in some situations. For example, this worked:

```md
[link to Dockerfile](Dockerfile.ziti-host)
```

While this did not work:

```md
[link to Dockerfile](./Dockerfile.ziti-host)
```